release-1.6/release-notes-draft.md

```diff
@@ -53,8 +53,49 @@
 ### New Features
 - Add new command "kubectl set selector" ([#38966](https://github.com/kubernetes/kubernetes/pull/38966), [@kargakis](https://github.com/kargakis))
 - Add kubectl create poddisruptionbudget command ([#36646](https://github.com/kubernetes/kubernetes/pull/36646), [@kargakis](https://github.com/kargakis))
+- Add new DaemonSet status fields to kubectl printer and describer.  ([#42843](https://github.com/kubernetes/kubernetes/pull/42843), [@janetkuo](https://github.com/janetkuo))
```

Comment:
I don't know if it makes sense to try and maintain a strict mapping between "new features" and "features appearing in `kubernetes/features`, @idvoretskyi. I think we should give broad deference to the SIG contributors and we can move any thing around ourselves if we need to make edits on their organization